index.html

```diff
@@ -9,7 +9,7 @@
 
   <body id="body" style="text-align: center">
     <h1> FREECODECAMP SURVEY FORM</h1>
-    <h2>Thank you for taking part.</h2>
+    <h1>Thank you for taking part.</h1>
```

Comment:
Hi, kindly use only one h1 semantic tag for your project as this shows the main heading for the whole project.
Thanks🙂